tutorials/beginner/1sttimesetup.md

```diff
@@ -0,0 +1,22 @@
+---
+layout: tutorial-beginner
+---
+
+{% include base.html %}
+
+# First-time setup of openHAB2
```

Comment:
I would suggest to replace all occurrences of "openHAB2" and "openHAB 2" by simply "openHAB".